src/Compiler/CompilerOptions.cpp

```diff
@@ -208,6 +208,11 @@ llvm::cl::opt<bool> enableSimdDataLayout("simd-data-layout",
                    "Set to 'true' if you want to enable SIMD optimizations."),
     llvm::cl::init(false), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::opt<bool> enablePatternShapeInference("pattern-shape-inference",
+    llvm::cl::desc("Enable pattern based shape inference (default=false)\n"
+                   "Set to 'true' to enable."),
+    llvm::cl::init(false), llvm::cl::cat(OnnxMlirCommonOptions));
```

Comment:
I tried `init(true)` and verified that `cmake --build . --target check-onnx-lit` passed